idl/attest.idol

```diff
@@ -0,0 +1,39 @@
+// Interface to Root of Trust for Reporting (ROT-R) task.
+
+Interface(
+    name: "Attest",
+    ops: {
```

Comment:
I should get around to making this the default, but could I encourage you to apply
```
encoding: Hubpack,
```
to each operation in any new IDL interface? It will let you use more flexible argument and error types.